### ISSUE TITLE
Automation of SSTV audio to images

### ISSUE THREAD

**Issue by KiwiInNZ** [wxcapture/wxcapture#7]
Originally the receive_sstv.py code was just capturing the audio from a pass.

This was updated to generate the SSTV images using QSSTV.

However QSSTV is not automatically started, which will result in the SSTV decoding failing.

Need to identify how QSSTV can be automatically started after a server reboot.

qsstv.py is the basis for this code, but it currently needs to be run from a VNC terminal window in the background manually after a server reboot.

**Comment by technobird22** [comment]
Setting to a low priority for now as it does not look like there are any ISS SSTV events coming up.

**Comment by KiwiInNZ** [comment]
This is blocked by how QSSTV can be automatically started when a Pi boots to process audio.

**Comment by KiwiInNZ** [comment]
Closing as not feasible to implement using QSSTV
